README.md

```diff
@@ -111,12 +111,22 @@ DLLs are libraries of vendor files that are referenced by other tasks.  This spe
 
 #### BASE_URL
 
-The `BASE_URL` defines the root path where the bundled files will be located.  If you are uploading to a non-root path on a server (`http://myserver.com/this-path`) this will need to be defined.
+The `BASE_URL` defines the root path where the bundled files will be located.  If you are uploading to a non-root path on a server (`http://myserver.com/this-path`) this will need to be defined. This can be done on a one-off basis via build commands, or permanently via `package.json`.
```

Comment:
Maybe add something that explains when you're fine with using the defaults? Like, "If you are uploading to a non-root path on a server (`http://myserver.com/this-path`) this will need to be defined. For local development or [something something], the default path of `/` should be sufficient..."

Reply:
I've updated the readme to cover when using the default `BASE_URL` (`/`) is appropriate and tried to word it in a way that also adds more clarity to what the property actually does.